spec/features/users/user_edit_spec.rb

```diff
@@ -35,7 +35,7 @@
     other = FactoryGirl.create(:user, email: 'other@example.com')
```

Comment:
`FactoryGirl.` prefix can be removed from all specs thanks to `FactoryGirl::Syntax::Methods` included by https://github.com/xudongcamsys/homefoodie/blob/master/spec/support/factory_girl.rb